### PR TITLE
Progress update october

### PR DESCRIPTION
**Pull request by davegson**

### CODE REVIEW

File: _data/next.yml [hunk 1]
```diff
@@ -16,38 +16,37 @@
...
+        - Website
       team:
-        - Daniel
-    - title: Administration Interface
+        - David
+    - title: "Inception: Reaching Out"
       description:
-        - Enable admins to easily accept cash payments, manage refunds or suspend rogue users.
-        - Also provide some basic insights on signups, payments, survey results, etc...
+        - As the SPN will saw the light of day, the time has come to spread the word, also by spending money.
```
Comment (Raphty) [inline _data/next.yml:31]: also by spending money. -> via word of mouth and buying advertisements.

das sollte expliziter sein, wir bestechen niemanden.

File: _posts/2020-11-02-progress-update-october.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        - type: 'h3'
+          text: "Backend Done, Frontend Task Moved to Backlog"
+        - type: p
+          text: "The server part has been done for a while now, which is great! As a result the task to implement the client side feature will be very straightforward. We intended to push this feature directly with the redesigned UI but decided against it to not slow the release down. So off to the backlog it goes."
+        - type: a
+          text: "Dive into our Backlog"
+          href: "https://safing.io/backlog/"
+    - title: "Inception: Reaching Out"
+      description:
+        - As the SPN will saw the light of day, the time has come to spread the word, also by spending money.
```
Comment (Raphty) [inline _posts/2020-11-02-progress-update-october.html:43]: same as above

File: _data/next.yml (hunk 1 above)
Comment (davegson) [reply _data/next.yml:31]: what about: "via word of mouth and paid marketing"?

File: _posts/2020-11-02-progress-update-october.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+          text: "Structure Set, Texts Are Being Written"
+        - type: p
+          text: "The Portmaster is a powerful tool, and hence it is complex. Showing and documenting this in an easy fashion is a challenge, so David put a lot of thought into finding a big-picture structure with which the docs should be easy to understand and navigate. With that being done, now the texts for each part, module and component are being written."
+        - type: 'h3'
+          text: "Design Is Still Primitive And Needs Work"
+        - type: p
+          text: "Since Luke is fully invested into the Portmaster Redesign we are just using the very outdated docs design for now. We might release this project in different stages so you get content fast and then make it fancier and more informative as we go."
+
+    - title: Cryptocurrency Payments
+      description: Enable you to pay for the SPN with cryptocurrencies such as Bitcoin, Ethereum, Zcash and Monero
```
Comment (Raphty) [inline _posts/2020-11-02-progress-update-october.html:150]: wir machen mal KEIN Zcash, nur BTC ETH XMR

File: _data/next.yml (hunk 1 above)
Comment (Raphty) [reply _data/next.yml:31]: ja passt

**Review by Raphty — APPROVED** [review]

**Pull request merged**